registry.json

```diff
@@ -4221,5 +4221,20 @@
     "telegram": "saganpool",
     "operatorName": "John",
     "github": ""
+  },
+  {
+    "ticker": "CBITS",
+    "poolId": "6782865c91a230fb379a433a0630a68a9adcdca9e518ad73302ef805",
+    "social": {
+      "website": "https://cardanobits-pool.com/",
+      "twitter": "MavidDuster",
+      "discord": "MavidDuster#8325",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "Mavid Duster",
```

Comment:
Please can you provide your telegram handle? This should be the username found in Settings -> Profile -> Username